src/System.Management.Automation/FormatAndOutput/common/TableWriter.cs

```diff
@@ -227,14 +241,28 @@ internal void GenerateRow(string[] values, LineOutput lo, bool multiLine, ReadOn
                 foreach (string line in GenerateTableRow(values, currentAlignment, lo.DisplayCells))
                 {
                     generatedRows?.Add(line);
-                    lo.WriteLine(line);
+                    if (ExperimentalFeature.IsEnabled("PSAnsiRendering") && isHeader)
+                    {
+                        lo.WriteLine(PSStyle.Instance.Formatting.TableHeader + line + PSStyle.Instance.Reset);
```

Comment:
I wonder if we shouldn't be creating a new method in `LineOutput` which allows for additional attributes. Again, nothing to do in this PR but we should probably be thinking about it.

Reply:
Should this feature be moved out of experimental, then the new code would replace the old one so all the `if..else` will be collapsed.